### DISCUSSION TITLE
[Android] How to control the display and hide of FilamentInstance, or to control rendering or not renderin

### DISCUSSION THREAD

**Discussion by fuchao** [google/filament#5684] [Q&A] [answered]
I need to render a lot of FilamentInstance dynamically, sometimes according to the data to render or not to render a
FilamentInstance, how can I control their state, thank you.

**Comment by romainguy** [comment, marked as answer]
You can change the visibility of a `Renderable` using `setLayerMask` on `RenderableManager`. A `FilamentInstance` has a list of entities, you need to change the layer mask on those entities.

**Reply by fuchao** [reply]
Thank you so much!
